.flake8

```diff
@@ -6,3 +6,4 @@ exclude =
 max-line-length = 121
 ignore =
     F401,F403 penny_university/settings/
+    W503, W504
```

Comment:
I'm ignoring lint complaints about splitting lines on boolean operators. This is just too useful when I'm dealing with long `if` clauses.